app/src/main/kotlin/com/wire/android/ui/main/MainDrawer.kt

```diff
@@ -0,0 +1,159 @@
+package com.wire.android.ui.main
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.DrawerValue
+import androidx.compose.material.ScaffoldState
+import androidx.compose.material.Text
+import androidx.compose.material.rememberDrawerState
+import androidx.compose.material.rememberScaffoldState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.rememberNavController
+import com.wire.android.R
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+@Composable
+fun Drawer(scope: CoroutineScope, scaffoldState: ScaffoldState, navController: NavController) {
+
+    val topItems = listOf(MainScreen.Conversations, MainScreen.Archive, MainScreen.Vault)
+    val bottomItems = listOf(MainScreen.Settings, MainScreen.Support)
+
+    Column(
+        modifier = Modifier
+            .padding(top = 40.dp, start = 8.dp, end = 8.dp, bottom = 16.dp)
+
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.ic_big_logo),
```

Comment:
maybe move it to some reusable component ? I guess it is going to be reused ? 
Also move contentDescription to String resource